ext/pcntl/tests/pcntl_unshare_04.phpt

```diff
@@ -1,21 +1,38 @@
 --TEST--
-pcntl_unshare() with wrong flag
+pcntl_unshare() - Error with wrong flag. Test return value type for each valid flag.
 --EXTENSIONS--
 pcntl
-posix
 --SKIPIF--
 <?php
 if (!function_exists("pcntl_unshare")) die("skip pcntl_unshare is not available");
 ?>
 --FILE--
 <?php
 
+$result = null;
+
 try {
-    pcntl_unshare(42);
+    $result = pcntl_unshare(-1);
 } catch (ValueError $exception) {
     echo $exception->getMessage() . "\n";
 }
 
+var_dump($result);
+
+foreach ([CLONE_NEWNS, CLONE_NEWIPC , CLONE_NEWUTS , CLONE_NEWNET, CLONE_NEWPID ,CLONE_NEWUSER, CLONE_NEWCGROUP] as $flag) {
+    $result = @pcntl_unshare($flag);
+    var_dump($result);
+    $result = null;
+}
+
 ?>
---EXPECT--
+--EXPECTF--
 pcntl_unshare(): Argument #1 ($flags) must be a combination of CLONE_* flags
+NULL
+bool(%s)
```

Comment:
That doesn't make sense to me. If we wanted to verify that the function returns a `bool`, we should output the type. But the function is already guaranteed to always return bool (there are respective checks in debug mode), so the testing this is superfluous.